apps/hpc-cdm-e2e/eslint.config.mjs

```diff
@@ -0,0 +1,4 @@
+import nxESlintPlugin from '@nx/eslint-plugin';
+import baseConfig from '../../eslint.config.mjs';
+
+export default [...baseConfig, ...nxESlintPlugin.configs['flat/react']];
```

Comment:
I copied from wrong file when suggesting changes in [this comment](https://github.com/UN-OCHA/hpc-cdm/pull/505#discussion_r2103106941). I correctly mentioned we should use Cypress plugin though.
```suggestion
import cypress from 'eslint-plugin-cypress/flat';
import baseConfig from '../../eslint.config.mjs';

export default [cypress.configs['recommended'], ...baseConfig];
```
Do the same in other E2E project.